### DISCUSSION TITLE
過敏真的好痛苦

### DISCUSSION THREAD

**Discussion by Jamesmoneya** [1101-GS4719-C/Python-Community-GS4719-C#1433] [General]
本人鼻過敏很嚴重，有時嚴重到喉嚨會癢
已經吃了很長時間的抗組織胺
但最近發現吃藥好像沒什麼用了
加上有時外出真的會忘記帶藥
有看到能施打脫敏針，一開始每週注射一次，3-6個月後達到維持，而後2-4週注射一次（網路上看是這樣）

想請問脫敏針要去哪裡打，問過診所他們只說有過敏針，但過敏針跟脫敏針是不是不一樣的啊⋯

**Comment by TW-Maverick** [comment]
我上網查了一下，引用文章中的這段話【一般中大型醫院之兒科或耳鼻喉科皆有提供減敏治療，部份兒科或耳鼻喉科之診所亦有提供此治療，由於需要長期且持續地針劑注射，一般建議以離家近的診所或醫院治療為主，以免長期奔波之苦】，我建議原po可以先打去比較大間的醫院問問看。

1. 聯新醫院
2. 天晟醫院
3. 部桃

像我家母也有過敏的問題，已經持續大約20年了，時不時都會流鼻涕跟鼻塞，如果是我的話，我連感冒3個禮拜都已經快要受不了了！
希望原po可以早日康復


🔗參考文章連結
[治療過敏救星 脫敏針](https://www.elcaminohealth.org/community/chinese-health-initiative/conditions-treatments/health-article-experts/allergy-shot-to-treat-allergy-zh)
[減敏療法](https://sp1.hso.mohw.gov.tw/doctor/Often_question/type_detail.php?q_type=%B4%EE%B1%D3%C0%F8%AAk)

**Comment by ALAN30359704** [comment]
雖然我比加不算是過敏的體質，但是我身邊有非常多的家人跟朋友都數過敏體質，常常在換季的時候會不停的打噴嚏跟咳嗽看了真的覺得他們非常的辛苦，但是我有聽過一個說法是在現在這個世代有非常多的過敏，而我認為跟遺傳還有人的演化說不定有一定的關係。
